WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -796,7 +791,9 @@ private void LogCoinJoinSummary(ImmutableArray<AliceClient> registeredAliceClien
 		int sameTxAllowance = GetRandomBiasedSameTxAllowance(rnd, percent);
 
 		var winner = new List<SmartCoin>();
+		winner.Add(selectedNonPrivateCoin);
 		foreach (var coin in finalCandidate
+			.Except(new [] {selectedNonPrivateCoin})
```

Comment:
Note this makes sure that `selectedNonPrivateCoin` participates, as that's what we want to mix the most.

Reply:
This is actually unrelated to this PR.